### PR TITLE
build: Force double quotes through prettier

### PR DESCRIPTION
**Pull request by fedemp**
Change prettier configuration so it prefers double quotes over single quotes.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

**Comment by elibarzilay** [comment]
(I prefer double quotes too, but changing the default seems like it's better if someone else looks at it...)

**Comment by fedemp** [comment]
> (I prefer double quotes too, but changing the default seems like it's better if someone else looks at it...)

`dtslint` actually mandates double quotes, so if you were to run prettier on the codebase, you would break everything. :)

**Comment by pkuczynski** [comment]
Why would we force dbl quotes over single quotes? What's the reasoning behind it?

**Comment by fedemp** [comment]
In before, I'm not a mantainer. 

> Why would we force dbl quotes over single quotes? What's the reasoning behind it?

To avoid having a flame war over double and single quotes, and because changing now the chose option would  modify the whole repository.

**Comment by pkuczynski** [comment]
@fedemp this change has been made already, so I am not sure if I understand you correctly? And it caused such PR like https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52790 - whats the point?

**Comment by fedemp** [comment]
Let me rephrase. `dtslint` forces the use of double quotes. It has been configured like that from the very beginning (I guess). I updated the `prettier` configuration so if a contributor runs `prettier` over the codebase, they will get valid code.

Prettier is not run as a check, so it's like double quotes are enforced by prettier.

**Comment by pkuczynski** [comment]
If `dtslint` would force dbl quotes, why for example this PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52850 didn't fail?

**Comment by pkuczynski** [comment]
@fedemp?

**Comment by fedemp** [comment]
> If `dtslint` would force dbl quotes, why for example this PR #52850 didn't fail? There is plenty of single quotes, yet the build runs smoothly.

A wizard did it? I must insist that I did not configured `dtslint`; I just made an outdated `prettier` configuration play well with the currently expected style. And also that `prettier` is not run as a check during the test process. 

````diff
diff --git c/types/ckeditor__ckeditor5-core/src/commandcollection.d.ts i/types/ckeditor__ckeditor5-core/src/commandcollection.d.ts
index bc9786a044..b954b4c4d9 100644
--- c/types/ckeditor__ckeditor5-core/src/commandcollection.d.ts
+++ i/types/ckeditor__ckeditor5-core/src/commandcollection.d.ts
@@ -1,4 +1,4 @@
-import Command from "./command";
+import Command from './command';
 
 export default class CommandCollection implements Iterable<[string, Command]> {
     constructor();
````
----
```shell
$ npm run test-all                                            

> definitely-typed@0.0.3 test-all
> node node_modules/@definitelytyped/dtslint-runner/dist/index.js --path .

Using local DefinitelyTyped at .
Parsing definitions...
Found 7674 packages.
Parsing in 4 child process(es)...
Parsing took 44.066 s
Running: git rev-parse --verify master
a84297d5fafa539c0181ef20dd399d650b0a363b
Running: git diff master --name-status
M	types/ckeditor__ckeditor5-core/src/commandcollection.d.ts
Testing 1 changed packages: ckeditor__ckeditor5-core
Testing 5 dependent packages: ckeditor__ckeditor5-core,ckeditor__ckeditor5-essentials,ckeditor__ckeditor5-export-pdf,ckeditor__ckeditor5-export-word,ckeditor__ckeditor5-ui
Installing NPM dependencies...
1> ckeditor__ckeditor5-core START
2> ckeditor__ckeditor5-core START
3> ckeditor__ckeditor5-essentials START
4> ckeditor__ckeditor5-export-pdf START
4> ckeditor__ckeditor5-export-pdf OK
4> ckeditor__ckeditor5-export-word START
3> ckeditor__ckeditor5-essentials OK
3> ckeditor__ckeditor5-ui START
4> ckeditor__ckeditor5-export-word OK
3> ckeditor__ckeditor5-ui OK
1> ckeditor__ckeditor5-core failing:
1> Error: /home/federico/DefinitelyTyped/types/ckeditor__ckeditor5-core/src/commandcollection.d.ts:1:21
1> ERROR: 1:21  quotemark  ' should be "
1> 
1>     at /home/federico/DefinitelyTyped/node_modules/dtslint/bin/index.js:207:19
1>     at Generator.next (<anonymous>)
1>     at fulfilled (/home/federico/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)
2> ckeditor__ckeditor5-core failing:
2> Error: /home/federico/DefinitelyTyped/types/ckeditor__ckeditor5-core/src/commandcollection.d.ts:1:21
2> ERROR: 1:21  quotemark  ' should be "
2> 
2>     at /home/federico/DefinitelyTyped/node_modules/dtslint/bin/index.js:207:19
2>     at Generator.next (<anonymous>)
2>     at fulfilled (/home/federico/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)


=== SUGGESTIONS ===

{"ckeditor__ckeditor5-core": [{"fileName":"/home/federico/DefinitelyTyped/types/ckeditor__ckeditor5-core/index.d.ts","ruleName":"npm-naming","message":"The declaration doesn't match the JavaScript module 'ckeditor__ckeditor5-core'. Reason:\nThe JavaScript module exports a property named 'PendingActions', which is missing from the declaration module."},{"fileName":"/home/federico/DefinitelyTyped/types/ckeditor__ckeditor5-core/index.d.ts","ruleName":"npm-naming","message":"The declaration doesn't match the JavaScript module 'ckeditor__ckeditor5-core'. Reason:\nThe JavaScript module exports a property named 'icons', which is missing from the declaration module."}]}


=== PERFORMANCE ===

{ "architect__functions": 14541, "ckeditor__ckeditor5-engine": 12856, "ckeditor__ckeditor5-utils": 12831, "aws-serverless-express": 12804, "koa-ratelimit": 12757, "ckeditor__ckeditor5-ui": 12711, "bull-arena": 12409, "redlock": 12298, "connect-redis": 12236, "rate-limit-redis": 12160, "aws-lambda": 12144, "ckeditor__ckeditor5-autosave": 12054, "ckeditor__ckeditor5-core": 11853, "ckeditor__ckeditor5-export-pdf": 11747, "ckeditor__ckeditor5-export-word": 11742, "ckeditor__ckeditor5-essentials": 11711, "apex.js": 11321, "ioredis": 10660, "bull": 10457, "fastify-rate-limit": 10216, "ratelimiter": 9957, "cache-manager-ioredis": 9836, "hapi__catbox-redis": 9816, "google-adwords-scripts": 9247, "google-apps-script-oauth2": 8321, "google-apps-script": 5608, "nut-pipe": 3500, "lambda-tester": 3489, "lambda-wrapper": 3449, "cfn-response": 3277, "cfn-response-promise": 3276, "hyphen": 2237}
  * Percentiles: 
99: 14541
95: 12853.5
90: 12812.099999999999
70: 12158.4
50: 11516


=== ERRORS ===



Error in ckeditor__ckeditor5-core
Error: /home/federico/DefinitelyTyped/types/ckeditor__ckeditor5-core/src/commandcollection.d.ts:1:21
ERROR: 1:21  quotemark  ' should be "

    at /home/federico/DefinitelyTyped/node_modules/dtslint/bin/index.js:207:19
    at Generator.next (<anonymous>)
    at fulfilled (/home/federico/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)


Error in ckeditor__ckeditor5-core
Error: /home/federico/DefinitelyTyped/types/ckeditor__ckeditor5-core/src/commandcollection.d.ts:1:21
ERROR: 1:21  quotemark  ' should be "

    at /home/federico/DefinitelyTyped/node_modules/dtslint/bin/index.js:207:19
    at Generator.next (<anonymous>)
    at fulfilled (/home/federico/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)
```

**Comment by pkuczynski** [comment]
This probably happens, because you are using an old dtslint-runner, which `tslint.json` was extending `tslint:all`, which has `quotemark` set to `"double"`. Check here: https://github.com/DefinitelyTyped/dtslint-runner/blob/master/tslint.json

Please note that this repo has been archived and moved to a new location. The new version of this package does not provide any custom `tslint.json` (see https://github.com/microsoft/DefinitelyTyped-tools/tree/master/packages/dtslint-runner) and is basically pulling `dtslint`, which in config extends from `tslint:latest` (https://github.com/microsoft/dtslint/blob/master/tslint.json). This ruleset does not define `quotemark` (https://github.com/palantir/tslint/blob/master/src/configs/latest.ts), but extends from `tslint:recommended` (https://github.com/palantir/tslint/blob/master/src/configs/recommended.ts), which does not define `quotemark` either. So this rule should be disabled.

When I run `test-all` on my local, I am not getting any issues around quotes. Thats why also the build does not fail. You must be either using an old package or have some kind of invalid local configuration.

**Comment by pkuczynski** [comment]
I created #52946 which is reverting your change and still the build does not fail! This is because qutes do not matter for dtslint (proved above). So your change with prettier would cause anybody using prettier to create noisy PR containing " to ' changes. That's pointless. Prettier was configured for singlequotes since June 2019 (https://github.com/DefinitelyTyped/DefinitelyTyped/commit/a52799f9e9c606cd42297a7ebd3945187d3f9b3f#diff-979a4a52db5287c891eac28a141bfa97bf8b0732ed5afae848d90956ef80e78d) and I really don't see a reason for your change, which is causing more harm then good :(

**Comment by pkuczynski** [comment]
I would like to kindly ask you for approval for #52946 @fedemp

**Comment by fedemp** [comment]
> I would like to kindly ask you for approval for #52946 @fedemp

I'm not a mantainer.
